src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -366,18 +374,36 @@ private void ValidateEnumDependencies(JsonSchema schema, JObject map, List<Error
                             if (allowLists.TryGetValue(dependentFieldValue, out var allowList) &&
                                 Array.IndexOf(allowList, fieldValue) == -1)
                             {
-                                errors.Add(Errors.InvalidPairedAttribute(JsonUtility.GetSourceInfo(map), fieldName, fieldValue, dependentFieldName, dependentFieldValue));
+                                errors.Add(Errors.InvalidPairedAttribute(JsonUtility.GetSourceInfo(map), fieldName, fieldValue, dependentFieldName, dependentFieldValue), dependentFieldName);
                             }
                         }
                         else
                         {
-                            errors.Add(Errors.MissingPairedAttribute(JsonUtility.GetSourceInfo(map), fieldName, dependentFieldName));
+                            errors.Add(Errors.MissingPairedAttribute(JsonUtility.GetSourceInfo(map), fieldName, dependentFieldName), fieldName);
                         }
                     }
                 }
             }
         }
 
+        private Error OverwriteError(JsonSchema schema, string name, Error baseError)
+        {
+            if (!string.IsNullOrEmpty(name) && schema.OverwriteErrors.TryGetValue(name, out var attributeOverwriteErrors) && attributeOverwriteErrors.TryGetValue(baseError.Code, out var overwriteError))
+            {
+                return new Error(
+                    overwriteError.Severity == ErrorLevel.Off ? baseError.Level : overwriteError.Severity,
```

Comment:
I should be able to override level to off, consider make Severity an `ErrorLevel?`

```suggestion
                    overwriteError.Severity is null ? baseError.Level : overwriteError.Severity,
```